src/joAbstractLinearOperator/extra_functions.jl

```diff
@@ -20,6 +20,83 @@ iswide(A::joAbstractLinearOperator{DDT,RDT}) where {DDT,RDT} = (A.m < A.n)
 # iscomplex(jo)
 iscomplex(A::joAbstractLinearOperator{DDT,RDT}) where {DDT,RDT} = !(DDT<:Real && RDT<:Real)
 
+# iscomplex(jo)
+iscomplex_b(A::joAbstractLinearOperator{DDT,RDT}) where {DDT,RDT} = !(DDT<:Real && RDT<:Real)
+
+
+"""
+    normest(jo;numiters, tol)
+Approximate spectral norm (largest singular value) through power iterations. For small matrices a single iteration 
+should be enough. 
+"""
+
+function normest(A::joAbstractLinearOperator{DDT,RDT};numiters::Integer=5,tol::Float64=0.,verbose::Bool=false) where {DDT,RDT}   
+
+    spec_norm_aprox = 0
+
+    u_hat = DDT<:Real ? jo_convert(DDT,randn(A.m)) : jo_convert(DDT,complex.(randn(A.m),randn(A.m)))
+    u_hat  = u_hat / norm(u_hat,2)
+    v_hat  = deepcopy(u_hat)
```

Comment:
The is not point in deepcopy since you overwrite it later. And it is wrong anyway.